projects/components/src/datagrid/datagrid.component.ts

```diff
@@ -233,6 +239,9 @@ export class DatagridComponent<R> implements OnInit, AfterViewInit {
     ContextualButtonPosition = ContextualButtonPosition;
     GridColumnHideable = GridColumnHideable;
     TooltipSize = TooltipSize;
+    DEFAULT_CLIPTEXT: CliptextConfig = {
```

Comment:
Does this really need to be a property on the class? I think just an object outside the class is fine, and it won't require `this.`